guide/src/main/asciidoc/api_specifications.adoc

```diff
@@ -14,17 +14,17 @@ In this case, special attention should be given that the API specifications rema
 [[openapi]]
 === OpenAPI (Swagger)
```

Comment:
Should we drop "(Swagger)" from the title?

Reply:
done